content/en/blog/_posts/2023-09-15-userns-stateful-pods/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kuberntes 1.28: Alpha support to run stateful pods too!"
+date: 2023-09-15
+slug: userns-alpha
+---
+
+**Authors:** Rodrigo Campos Catelin (Microsoft), Giuseppe Scrivano (Red Hat), Sascha Grunet (Red Hat)
+
+Kubernetes v1.25 introduced support for user namespaces for only stateless
+pods. In Kubernetes 1.28 we lifted that restriction, after some design changes
+were done in 1.27.
+
+The beauty of this feature is that:
+ * it is trivial to adopt (we need just to set a bool in the pod spec)
+ * doesn't need any changes for **most** applications
+ * improves security for **all** Kubernetes users by  _drastically_ enhancing
+   the isolation of containers and mitigating CVEs rated HIGH and CRITICAL.
```

Comment:
All CVEs with those ratings? Also, really **all** users? Even people who only run Windows Pods?

Reply:
Removed the "for all kubernetes users" part